pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -390,6 +390,24 @@ type AlertmanagerGlobalConfig struct {
 
 	// The default Pagerduty URL.
 	PagerdutyURL *string `json:"pagerdutyUrl,omitempty"`
+
+	// The default Telegram API URL.
+	// Default value will be "https://api.telegram.org"
+	// +optional
+	TelegramAPIURL *string `json:"telegramApiUrl,omitempty"`
```

Comment:
Let's respect [Kubernetes conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions) on field naming. We're not being consistent in the past but I'd rather follow them from now on.
```suggestion
	TelegramAPIURL *string `json:"telegramAPIURL,omitempty"`
```